engine/engine.go

```diff
@@ -518,10 +519,21 @@ func createEngine(terragruntOptions *options.TerragruntOptions) (*proto.EngineCl
 
 	terragruntOptions.Logger.Debugf("Creating engine %s", localEnginePath)
 
+	engineLogLevel := os.Getenv(EngineLogLevelEnv)
```

Comment:
It will not work with, `false`, `0`, and will not notify about a wrong value. The same with `EngineCachePathEnv`, `TG_ENGINE_SKIP_CHECK`.

In this particular case I would recommend adding `opts.DisableLog`

```go
		&cli.BoolFlag{
			Name:   TerragruntLogDisableFlagName,
			EnvVar: TerragruntLogDisableEnvName,
			Destination: &opts.DisableLog,
```

Let's agree that the only entry point for **_all_** environment variables is CLI flags, this will help avoid issues in the future, when we forget about the existence of this "orphan", and it will live its own life. For example, when globally renaming a prefix or redesign the CLI interface. If for some reason you don't need to show the evn variable flag in the CLI help, you can simply hide it from users with `Hidden: true`.